content/en/docs/contribute/new-content/open-a-pr.md

```diff
@@ -167,7 +167,7 @@ Figure 2. Working from a local fork to make your changes.
 1. In a terminal window, clone your fork and update the [Docsy Hugo theme](https://github.com/google/docsy#readme):
 
    ```shell
-   git clone git@github.com/<github_username>/website
+   git clone git@github.com:<github_username>/website
```

Comment:
Since this command is using the SSH method of cloning, shouldn't `.git` be added at the end (as seen in the URL of the repo we are trying to clone) ? 
```suggestion
   git clone git@github.com:<github_username>/website.git
```